### PR TITLE
co-authored-by: Jeff Grunewald <jeffrey.a.grunewald@accenture.com>

### PR DESCRIPTION
**Pull request by bbalser**

### CODE REVIEW

**Comment by bbrewer-papa-x** [comment]
👀

**Comment by jakeprem** [comment]
Better than RailStream :steam_locomotive:

File: lib/reaper/feed_supervisor.ex [hunk 1]
```diff
@@ -63,12 +63,27 @@ defmodule Reaper.FeedSupervisor do
...
       %{
         id: cache_name,
         restart: restart_policy,
         start: {Cachex, :start_link, [cache_name, [limit: cache_limit]]}
-      },
+      }
+    ]
+  end
+
+  defp cache_spec(_reaper_config, _cache_name, _cache_limi, _restart_policy), do: []
```
Comment (jmorris-pillar) [inline lib/reaper/feed_supervisor.ex]: %s/limi/limit/g

File: test/unit/test_helper.exs [hunk 2]
```diff
@@ -29,3 +29,32 @@ defmodule TestHelper do
     assert_receive {:DOWN, ^ref, _, _, _}
   end
 end
+
+defmodule TempEnv do
```
Comment (jakeprem) [inline test/unit/test_helper.exs]: It might be helpful to add some documentation to this macro

File: test/integration/performance_test.exs [hunk 3]
```diff
@@ -102,7 +102,7 @@ defmodule Reaper.PerformanceTest do
     path
   end
 
-  defp generate_record(count) do
+  defp generate_record(_count) do
```
Comment (ManApart) [inline test/integration/performance_test.exs]: Why do we have this param if we don't use it?

**Pull request merged**